docs/containers.conf.5.md

```diff
@@ -300,6 +300,9 @@ pod consumes one lock.  The default number available is 2048.  If this is
 changed, a lock renumbering must be performed, using the
 `podman system renumber` command.
 
+**pull_type**="always"|"missing"|"never"
+Pull image before running a container. The default is **missing**.
```

Comment:
Please explain what these values mean. They can be interpreted but being explicit avoids potential confusion.